inst/WORDLIST

```diff
@@ -1,18 +1,27 @@
-
-altExp
 Alevin
+altExp
+altexp
```

Comment:
I'm curious where both of these are sneaking in... I feel like it should just be `altExp`?